templates/crd-serviceresolvers.yaml

```diff
@@ -1,8 +1,10 @@
+
```

Comment:
Why is the file `serviceresolvers`? Makes more sense to me to be crd-serviceresolver since this is the CRD for "ServiceResolver"

Reply:
This is how the new version of controller-tools generates it, so we decided to keep it as it's generated.

Reply:
ah okay, sounds good